src/SomeClass.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class SomeClass {

```

Comment:
It isn't related to generics, but if you are designing methods which do something, it's useful to return a result of the method execution, if possible. I mean if you count even numbers then return that count. And so on :)

Reply:
I thought that if the return statement is really necessary, you would mentioned this in the task, so I allowed myself to do it in another way
but I understand, that return it's useful, so in the future I will use it by default

Reply:
It's okay, I just wanted to be sure you understand it.